components/lib/carousel/Carousel.vue

```diff
@@ -4,90 +4,94 @@
-                    </slot>
-                </button>
-            </div>
-            <ul v-if="totalIndicators >= 0 && showIndicators" ref="indicatorContent" :class="[cx('indicators'), indicatorsContentClass]" @keydown="onIndicatorKeydown" v-bind="ptm('indicators')">
-                <li v-for="(indicator, i) of totalIndicators" :key="'p-carousel-indicator-' + i.toString()" :class="cx('indicator', { index: i })" v-bind="ptm('indicator', getIndicatorPTOptions(i))" :data-p-highlight="d_page === i">
                     <button
-                        :class="cx('indicatorButton')"
+                        v-if="showNavigators"
+                        v-ripple
                         type="button"
-                        :tabindex="d_page === i ? '0' : '-1'"
-                        :aria-label="ariaPageLabel(i + 1)"
-                        :aria-current="d_page === i ? 'page' : undefined"
-                        @click="onIndicatorClick($event, i)"
-                        v-bind="ptm('indicatorButton', getIndicatorPTOptions(i))"
-                    />
-                </li>
-            </ul>
+                        :class="cx('nextButton')"
+                        :disabled="forwardIsDisabled"
+                        :aria-label="ariaNextButtonLabel"
+                        @click="navForward"
+                        v-bind="{ ...nextButtonProps, ...ptm('nextButton') }"
+                        data-pc-group-section="navigator"
+                    >
+                        <slot name="nexticon">
+                            <component :is="isVertical() ? 'ChevronDownIcon' : 'ChevronRightIcon'" :class="cx('nextButtonIcon')" v-bind="ptm('nextButtonIcon')" />
+                        </slot>
+                    </button>
+                </div>
+                <ul v-if="totalIndicators >= 0 && showIndicators" ref="indicatorContent" :class="[cx('indicators'), indicatorsContentClass]" @keydown="onIndicatorKeydown" v-bind="ptm('indicators')">
+                    <li v-for="(indicator, i) of totalIndicators" :key="'p-carousel-indicator-' + i.toString()" :class="cx('indicator', { index: i })" v-bind="ptm('indicator', getIndicatorPTOptions(i))" :data-p-highlight="d_page === i">
+                        <button
+                            :class="cx('indicatorButton')"
+                            type="button"
+                            :tabindex="d_page === i ? '0' : '-1'"
+                            :aria-label="ariaPageLabel(i + 1)"
+                            :aria-current="d_page === i ? 'page' : undefined"
+                            @click="onIndicatorClick($event, i)"
+                            v-bind="ptm('indicatorButton', getIndicatorPTOptions(i))"
+                        />
+                    </li>
+                </ul>
+            </template>
+            <div v-else>
```

Comment:
You wrote in the documentation that an empty slot was added, but you did not make any changes about it.